### ISSUE TITLE
Ps vita freezes after exiting and restarting emulator 

### ISSUE THREAD

**Issue by Duhjoker** [SilicaAndPina/psvimgtools-frontend#81]
Hi there 

Sorry to bother you. I know your busy with your awesome work. But im having a glitch. 

Upon exiting VHBL and an emulator and then restarting vhbl and any other emulator including the one just used is causing my PS vita to freeze. 

Its huge problem since i cant use any of the system functions or buttons to turn the vita off. So far i have to put the console down amd wait 5-8 hours for the battery to dis-charge amd then i can use it again after a charge. 

I was wondering if any body else was having this problem and if so is there a fix?

**Comment by SilicaAndPina** [comment]
i dont think this is an issue with psvimgtools-frontend, but rather an issue with the emulator your using, but i am curious as to how its able to cause the system to crash? possibly an exploit could come from that.. but also possibly not it could totally just be a massive memory leak...

anyway if you hold down Power for like 30 seconds it should force power off :d
